.vscode/launch.json

```diff
@@ -5,24 +5,24 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": ".NET Core mode http (debug)",
+            "name": ".NET Core mode lsp (debug)",
```

Comment:
This is before I understood how this is working. Maybe we should remove those options completely?

Reply:
We probably want to create debugging targets for Lsp Tests project - it's easiest way to run and debug FSAC without using external client

Reply:
Ok I'll leave as is as I have no clue what you mean :D